examples/e-commerce/package.json

```diff
@@ -8,7 +8,6 @@
     "qs": "6.8.0",
     "react": "16.8.6",
     "react-compound-slider": "2.2.0",
-    "react-dom": "16.8.6",
-    "react-instantsearch-dom": "5.7.0"
```

Comment:
how is this imported now then?

Reply:
It is using an alias in the webpack configuration https://github.com/algolia/react-instantsearch/pull/2823/files#diff-6420f9faefa3a9a37b0b32aa2332d22fR28-R39